packages/main/src/Select.ts

```diff
@@ -434,56 +442,56 @@ class Select extends UI5Element {
 				options[firstEnabledOptionIndex].selected = true;
 				options[firstEnabledOptionIndex]._focused = true;
 				this._selectedIndex = firstEnabledOptionIndex;
-				this._text = options[firstEnabledOptionIndex].textContent;
+				this._text = options[firstEnabledOptionIndex].textContent as string;
 			}
 		}
 
-		this._syncedOptions = syncOpts;
+		this._syncedOptions = syncOpts as Array<Option>;
 	}
 
 	_enableFormSupport() {
-		const FormSupport = getFeature("FormSupport");
-		if (FormSupport) {
-			FormSupport.syncNativeHiddenInput(this, (element, nativeInput) => {
-				nativeInput.disabled = element.disabled;
+		const formSupport = getFeature<typeof FormSupport>("FormSupport");
+		if (formSupport) {
+			formSupport.syncNativeHiddenInput(this, (element: IFormElement, nativeInput: HTMLInputElement) => {
+				nativeInput.disabled = !!element.disabled;
 				nativeInput.value = element._currentlySelectedOption ? element._currentlySelectedOption.value : "";
```

Comment:
I would prefer to use this one instead of enhancing the interface since _currentlySelectedOption is not relevant to any other input control.

```ts
const selectElement = (element as Select)
nativeInput.disabled = !!element.disabled;
nativeInput.value = selectElement._currentlySelectedOption ? selectElement._currentlySelectedOption.value : "";
```